tubular/nominal.py

```diff
@@ -429,7 +429,7 @@ def transform(self, X):
         return X
 
 
-class MeanResponseTransformer(BaseNominalTransformer, BaseMappingTransformMixin):
+class MeanResponseTransformer(BaseNominalTransformer):
```

Comment:
I know it's off topic but there's an important typo in line 444 - n>1 should read n =1

Reply:
resolve din 48aa1075cd59809aa0b918791a82d78078626182